SQL/0000-00-05-ElectrophysiologyTables.sql

```diff
@@ -201,6 +200,87 @@ CREATE TABLE `physiological_archive` (
     ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+-- SQL tables for BIDS derivative file structure
+-- Create physiological_annotation_file_type table
+CREATE TABLE `physiological_annotation_file_type` (
+    `FileType`        VARCHAR(20)   NOT NULL UNIQUE,
+    `Description` VARCHAR(255),
+    PRIMARY KEY (`FileType`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create physiological_annotation_file table
+CREATE TABLE `physiological_annotation_file` (
+    `AnnotationFileID`    INT(10)      UNSIGNED NOT NULL AUTO_INCREMENT,
+    `PhysiologicalFileID` INT(10)      UNSIGNED NOT NULL,
+    `FileType`            VARCHAR(20)  NOT NULL,
+    `FilePath`            VARCHAR(255) NOT NULL,
+    PRIMARY KEY (`AnnotationFileID`),
+    CONSTRAINT `FK_phys_file_ID`
+        FOREIGN KEY (`PhysiologicalFileID`)
+        REFERENCES `physiological_file` (`PhysiologicalFileID`),
+    CONSTRAINT `FK_annotation_file_type`
+        FOREIGN KEY (`FileType`)
+        REFERENCES `physiological_annotation_file_type` (`FileType`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Create annotation_archive which will store archives of all the annotation files for
+-- Front-end download
+CREATE TABLE `physiological_annotation_archive` (
```

Comment:
If the annotations need to be synchronized with user inputs, I think this table should be deleted and the zip created on the fly when download is requested.

Reply:
This is a good point. This is the way that the pysiological files are handled in general, which is why I implemented it this way. You're right though that in this case, it does not make sense.

Reply:
For now, let's follow the example of the .tgz created by the pipeline specifically as a downloadable package. 
e.g. `bids_imports/Face13_BIDSVersion_1.1.0/sub-OTT166/ses-V1/eeg/sub-OTT166_ses-V1_task-faceO_eeg.tgz`
Basically the archive for annotations should be created/structured similarly. 

Agreed this should be revisited and discussed when we come to adding/editing annotations in the module - e.g. are all .tgz updated?

Reply:
Ok in that case I will leave this table in, as that is how it's structured for the physiological tables/files.